pytorch_lightning/metrics/sklearns.py

```diff
@@ -1103,9 +1032,16 @@ def forward(
         """
         # only return x and y here, since for now we cannot auto-convert elements of multiple length.
         # Will be fixed in native implementation
-        return np.array(super().forward(probas_pred=probas_pred,
-                                        y_true=y_true,
-                                        sample_weight=sample_weight)[:2])
+        return np.array(super().forward(probas_pred=probas_pred, y_true=y_true, sample_weight=sample_weight)[:2])
+
+    def aggregate(self, *tensors: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
+        """Aggregates results by stacking them instead of concatenating before averaging.
+
+        Returns:
+            the aggregated results
+        """
+        print(tensors)
+        return tuple([torch.stack(tmp).mean(0) for tmp in zip(*tensors)])
```

Comment:
```suggestion
        """
        return tuple([torch.stack(tmp).mean(0) for tmp in zip(*tensors)])
```
save the ink, save the trees!